taker-frontend/src/components/RadioCard.tsx

```diff
@@ -0,0 +1,38 @@
+import { Box, useColorModeValue, useRadio } from "@chakra-ui/react";
+import React from "react";
+
+export function RadioCard(props: any) {
+    const { getInputProps, getCheckboxProps } = useRadio(props);
```

Comment:
Imho you are misusing radio-groups to achieve _highlightable_ buttons. 

I had the same issue when creating the two toggle buttons `Long BTC` and `Short BTC` and opted for buttons as which I believe was recommended by react router. 

Anyway, there is a new example in place where I found the old one. Maybe this comes handy for you: 
https://reactrouter.com/docs/en/v6/examples/custom-link

Reply:
from chakra docs: "Please be aware that the example below should only be used if you really need
 a radio button for data collection purposes. If you want to toggle between
 different content on activation of a button use the Tabs component."

perhaps `Tabs` would work best, as this is exactly our use-case?

Reply:
I'd use radio buttons only in a form environment where the user has to select one item of a list (or if he can select multiple, use checkboxes). 
`Tabs` still sound a bit weird to me in this use case as it's not really a tab (as in the browser), imho it behaves more like buttons.

Reply:
> from chakra docs: "Please be aware that the example below should only be used if you really need
a radio button for data collection purposes. If you want to toggle between
different content on activation of a button use the Tabs component."

😅 Seems I missed that. 
I'll have a look how it is done for long / short and potentially just re-use that! Thanks for pointing me to that :)